python/housinginsights/ingestion/insertion_functions.py

```diff
@@ -0,0 +1,299 @@
+"""
```

Comment:
Ultimately we should probably refactor these to be part of the relevant cleaners, so they fit more closely with the existing code structure